src/Uno.UWP/WinRTFeatureConfiguration.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Uno
+{
+	public static class WinRTFeatureConfiguration
+	{
+		/// <summary>
+		/// Used by tests cleanup to restore the default configuration for other tests!
+		/// </summary>
+		internal static void RestoreDefaults()
+		{
+			GestureRecognizer.RestoreDefaults();
+		}
+
+		public static class GestureRecognizer
+		{
+			internal static void RestoreDefaults()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826514)